generators/app/templates/src/config/ReactotronConfig.ejs

```diff
@@ -3,10 +3,15 @@ import Reactotron, { trackGlobalErrors } from 'reactotron-react-native';
 import apisaucePlugin from 'reactotron-apisauce';
 import { reactotronRedux } from 'reactotron-redux';
 
+// If you want to use a physical device and connect it to reactotron, execute first `adb reverse tcp:9090 tcp:9090`
+let scriptHostname;
 if (__DEV__) {
+  const scriptURL = NativeModules.SourceCode.scriptURL;
+  scriptHostname = scriptURL.split('://')[1].split(':')[0];
   Reactotron.configure({ name: '<%= projectName %>' })
     .use(trackGlobalErrors())
     .use(apisaucePlugin())
     .use(reactotronRedux({ onRestore: state => Immutable(state) }))
     .connect();
 }
+Reactotron.configure({ host: scriptHostname }).connect();
```

Comment:
this line is duplicated with line 11. We're already executing configure and connect.

Reply:
will add it to previous line